packages/elements/src/color-picker/__demo__/index.html

```diff
@@ -39,21 +39,23 @@
         Default Hex Value (Optional): <input id="hexInputDefault" class="hex-input" value=""> 
       </div>
       <div>
-        <p> Output: </p>
+        <p> Output: <span id="readbleColor"></span></p>
         <textarea readonly id="defaultColorOutput"></textarea>
       </div>
       <script>
         // Default Color Picker
         const defaultColorPicker = document.getElementById('defaultColorPicker');
         const hexInputDefault = document.getElementById('hexInputDefault');
         const defaultColorOutput = document.getElementById('defaultColorOutput');
+        const readbleColor = document.getElementById('readbleColor');
         hexInputDefault.addEventListener('change', event => {
           defaultColorPicker.setAttribute('value', hexInputDefault.value);
         });
         defaultColorPicker.addEventListener('value-changed', event => {
           const value = event.detail.value;
           hexInputDefault.value = value;
           defaultColorOutput.value += 'Value Change : ' + value + '\n';
+          setTimeout(() => readbleColor.textContent = value ? defaultColorPicker._colorAriaLabel : '', 0);
```

Comment:
Can we get label from attribute instead of accessing private property ? Why does it have to be in `settimeout` ?

Reply:
**Can we get label from attribute instead of accessing private property ?**
- Sound good, I will try this one.

**Why does it have to be in settimeout ?**
- Correct me if I am wrong. aria-label works with `TranslatePromise` as an async task and will not translate immediately. That means it should wait with settimeout.